state.go

```diff
@@ -886,6 +896,14 @@ func (m *Memberlist) aliveNode(a *alive, notify chan struct{}, bootstrap bool) {
 			},
 			State: stateDead,
 		}
+		if len(a.Vsn) > 5 {
```

Comment:
Is this the place where the versions were getting zeroed due to not setting them?